internal/constellation/applyinit.go

```diff
@@ -85,12 +85,17 @@ func (a *Applier) Init(
 	// Create a wrapper function that allows logging any returned error from the retrier before checking if it's the expected retriable one.
 	serviceIsUnavailable := func(err error) bool {
 		isServiceUnavailable := grpcRetry.ServiceIsUnavailable(err)
-		a.log.Debug(fmt.Sprintf("Encountered error (retriable: %t): %s", isServiceUnavailable, err))
+		a.log.Debug(fmt.Sprintf("Encountered error (retriable: %t): %q", isServiceUnavailable, err))
 		return isServiceUnavailable
 	}
 
 	// Perform the RPC
-	a.log.Debug(fmt.Sprintf("Making initialization call, doer is %+v", doer))
+	a.log.Debug(fmt.Sprintf(
+		`Making initialization call, doer has the following parameters:
+  endpoint: %q
+  req.KmsUri: %q
+  req.StorageUri: %q
+`, doer.endpoint, doer.req.KmsUri, doer.req.StorageUri))
```

Comment:
```suggestion
	a.log.Debug("Making initialization call", "endpoint", doer.endpoint, "kmsUri", doer.req.KmsUri, "storageUri")
```

We should just use log attributes to log this information